### PR TITLE
[10.0][MIG] account_payment_show_invoice

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

**Review by rami-wafaie — APPROVED** [review]
Review + functional testing
:+1:

**Comment by rami-wafaie** [comment]
@pedrobaeza Can you please have a look at this?

File: account_payment_show_invoice/models/account_payment.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ModelName(models.Model):
```
Comment (adrienpeiffer) [inline account_payment_show_invoice/models/account_payment.py]: s/ModelName/AccountPayment

**Review by adrienpeiffer — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
@adrienpeiffer can you update your review?

**Review by adrienpeiffer — APPROVED** [review]
:+1:

**Pull request merged**